src/main/pages/che-7/overview/assembly_introduction-to-eclipse-che.adoc

```diff
@@ -17,40 +17,40 @@ summary:
-* Support of most recent tooling protocols: Language Server Protocol, Debug Adapter Protocol.  
-* Plug-in mechanism compatible with Visual Studio Code extensions.  
-* An SDK for creating custom cloud developer platforms.  
+factories
+* Pre-built stacks with the ability to create custom stacks for any language or runtime  
+* Enterprise integration via Keycloak for AD/LDAP  
+* Browser-based IDEs; integration with Che-Theia or any other web IDE (like Jupyter) 
+* Support of most recent tooling protocols: Language Server Protocol, Debug Adapter Protocol
+* Plug-in mechanism compatible with Visual Studio Code extensions
+* An SDK for creating custom cloud developer platforms
 
 
 [id="getting-started-with-che"]
 == Getting started with Che
 
 Get started with Eclipse Che by:
 
-* Following the link:{site-baseurl}che-7/running-che-locally[quick-start guide] to get Eclipse Che installed locally on your computer.
-* Learning more about Eclipse Che: link:{site-baseurl}che-7/what-is-eclipse-che[What is Eclipse Che] and link:{site-baseurl}che-7/high-level-che-architecture[Architecture overview]
-// TODO: link:che-features-and-benefits.html[Features and benefits of Eclipse Che].
-* Discovering Eclipse Che capabilities.
+* Following the link:{site-baseurl}che-7/running-che-locally[quick-start guide] to get Eclipse Che installed locally on your computer
+* Learning more about Eclipse Che: link:https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che/#what-is-eclipse-che[What is Eclipse Che] and link:{site-baseurl}che-7/high-level-che-architecture[Architecture overview]
+// TODO: link:che-features-and-benefits.html[Features and benefits of Eclipse Che]
+* Discovering Eclipse Che capabilities
 
 
 == Joining the community
 
 Interested in joining the community? Join us on the following channels:
 
-* *Public chat:* Join the public link:https://mattermost.eclipse.org/eclipse/channels/eclipse-che[Eclipse Che Mattermost] channel to chat with the developers.
-* *GitHub project repositories:* Report bugs, request features, and contribute in the main link:https://github.com/eclipse/che[Eclipse Che repository]. Improve documentation in the link:https://github.com/eclipse/che-docs[Eclipse Che docs repository].
-* *Support:* Visit StackOverflow to help other users of Eclipse Che: link:https://stackoverflow.com/questions/tagged/eclipse-che[Eclipse Che on StackOverflow].
-* *Community blog:* Learn about the latest of Eclipse Che and submit your blog posts to the link:https://medium.com/eclipse-che-blog[Eclipse Che blog].
-* *Bi-weekly meetings:* Join us in our link:https://github.com/eclipse/che/wiki/Che-Dev-Meetings[Che community meeting] every second Monday.
-* *Roadmap:* See the development link:https://github.com/eclipse/che/wiki/Roadmap[roadmap] on the wiki.
-
+* *Public chat:* Join the public link:https://mattermost.eclipse.org/eclipse/channels/eclipse-che[Eclipse Che Mattermost] channel to chat with the developers
```

Comment:
all of this was correct; periods in lists are not used when the item isn't a sentence; see IBM Style Guide pg. 62